base-buster/Containerfile

```diff
@@ -0,0 +1,9 @@
+FROM docker://debian:buster-slim
+
+COPY sources.list /etc/apt/sources.list
+RUN apt update; apt upgrade --yes
+RUN apt install --yes --no-install-recommends vim ca-certificates
+RUN apt-get clean
+RUN mkdir -p /opt/config /opt/storage /opt/log
```

Comment:
Maybe we should add ENV LANG C.UTF-8 or make otherwise sure that the locale is set, as we had problems with the locale settings in our old containers